src/types/system/statespace.jl

```diff
@@ -53,7 +53,7 @@ immutable StateSpace{T,S,M1,M2,M3,M4} <: LtiSystem{T,S}
     end
     dx[:] = A*x + B*ucalc
   end
-  function (sys::StateSpace)(t::Real, x::DEDataArray, dx::AbstractVector, u)
+  function (sys::StateSpace)(t::Real, x::DiffEqBase.DEDataArray, dx::AbstractVector, u)
```

Comment:
wasn't it working without the namespace resolution, `DiffEqBase.`? Sorry... :( Maybe we should be `using DiffEqBase: DEDataArray`?